util/geoJsonCenter.js

```diff
@@ -16,4 +20,4 @@ module.exports = function( geometry ){
   }
 
   return;
```

Comment:
Agreed. It's a semantic difference. `null` implies a result could not be computed, while `undefined` implies a variable wasn't initialized.